docs/https_source_interface.rst

```diff
@@ -4,45 +4,93 @@ HTTPS on the Source Interface
 The *Source Interface* for SecureDrop is automatically served over
 as a Tor Hidden Service, requiring a ``*.onion`` URL to access it.
 While Tor Hidden Services provide end-to-end encryption by default, as well
-as strong anonymity, high-profile organizations may wish to provide
-further verification to Sources that the SecureDrop instance belongs
-to the organization listed on the *Landing Page*.
+as strong anonymity, there are several reasons why you might want to consider
+deploying an additional layer of encryption and authentication via HTTPS:
+
+* The cryptographic primitives used by Tor Hidden Services are considered to be
+  outdated, and while there are no known compromises of the security of Tor
+  Hidden Services due to this issue, it may be wise to provide an additional
+  layer of transport encryption using stronger cryptographic primitives, which
+  is most easily achieved by setting up HTTPS on the Source Interface.
+
+  .. note:: This issue is being addressed by the Tor Project with their Next
+     Generation Onion Services design, but that design is still a work in
+     progress and is not expected to be deployed until December 2017 at the
+     soonest.
+
+* Extended Validation (EV) certificates, which are currently the only type of
+  certificates that may be issued for ``*.onion`` addresses, are intended to
+  attest to the identity of the organization running a service. This provides
+  an additional measure of authenticity (in addition to the organization's
+  *Landing Page* and the `SecureDrop Directory`_) to help assure sources that
+  they are communicating with the intended organization when they access a
+  given Source Interface.
```

Comment:
Even though the crypto in onion services is a bit aged, as you correctly state, in my view the reason why we want to allow HTTPS on the source interface is the second reason you describe here: to demonstrate to sources that they are communicating with the onion service that is indeed controlled by the news organization they want to talk to. 

While an additional layer of encryption is a nice bonus, if we really believed that there may be a break in onion service crypto in the wild and that this was a likely vector of attack, then we'd make HTTPS on all interfaces a requirement and not optional. Basically my concern is making sure that SecureDrop is not pushing difficult security decisions to users - in this case the security decision "is Tor onion service crypto sufficient for my purposes?" is not a question that most users are in a position to answer (but we are, and we answered that question yes). That said, I think we should remove the comments about onion service crypto but leave your clear description in the second point. 

Let me know your thoughts!

(Btw: I'm cool with putting an small OCSP note for full disclosure and because it will be gone soon in the next minor release, but generally I think we should just instruct users on what the right decision is and then make that the default)